.mockery.yaml

```diff
@@ -1,2 +1,11 @@
 ---
 disable-version-string: true
+outpkg: 'mocks'
+dir: '{{.InterfaceDir}}/mocks'
+packages:
+  go.woodpecker-ci.org/woodpecker/v2/woodpecker-go/client:
```

Comment:
do we need this or can we do this in `woodpecker-go/client` ?

Reply:
We need it. The inline mockery annotation is deprecated.